app/src/main/java/com/kickstarter/viewmodels/RewardsFragmentViewModel.kt

```diff
@@ -158,6 +179,12 @@ class RewardsFragmentViewModel {
                     }
         }
 
+        private fun differentShippingTypes(newRW: Reward, backedRW: Reward): Boolean =
```

Comment:
This is a little more readable. I normally only like to use the inline notation for one liners. Code that requires more than one line should be in a block.

```
private fun differentShippingTypes(newRW: Reward, backedRW: Reward): Boolean {
    if (newRW.id() != backedRW.id()) {
        return newRW.shippingType()?.toLowerCase() ?: "" != backedRW.shippingType()?.toLowerCase() ?: ""
    }
    return false
}
```

Reply:
I don't like the two returns but ok with it being in a block 👍